.circleci/verdaccio/config.yml

```diff
@@ -0,0 +1,27 @@
+storage: ./storage
+auth:
+  htpasswd:
+    file: ./htpasswd
+uplinks:
+  npmjs:
+    url: https://registry.npmjs.org/
```

Comment:
Not sure if it matters, but the lockfile seems to be using predominantly `https://registry.yarnpkg.com`, which I guess is also just a npmjs proxy.

Reply:
I think is just a CNAME https://twitter.com/1stvamp/status/1000174791129747456 but I have no clue how looks like now.

Reply:
For consistency, I changed the name but both should work the same way afaict